### ISSUE TITLE
Accessing Swagger UI causes crash

### ISSUE THREAD

**Issue by bravetarget** [wavesplatform/node-docker-image#1]
_This might be a bug more suited for `wavesplatform/Waves` but I've only experienced this behavior while running this docker image..._

**Describe the bug**
Navigating to the root API endpoint from a browser (which redirects to Waves REST API Swagger UI) will cause the node to crash.

**To Reproduce**
1. Run docker image with REST API fully enabled
2. From a browser navigate to the root of the API (i.e. `localhost:6869`)
3. It should redirect to Swagger UI, then crash the node

**Expected behavior**
Swagger UI works as intended, or give an option to completely disable the serving of this feature.

**Error Log**
```
Uncaught error from thread [wavesplatform-akka.actor.default-dispatcher-7]: javax/xml/bind/annotation/XmlRootElement, shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled for for ActorSystem[wavesplatform]
java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlRootElement
	at io.swagger.jackson.ModelResolver.resolve(ModelResolver.java:270)
	at io.swagger.jackson.ModelResolver.resolve(ModelResolver.java:185)
	at io.swagger.scala.converter.SwaggerScalaModelConverter.resolve(SwaggerScalaModelConverter.scala:86)
```

**Additional context**
Attempted on local docker daemon and remote kubernetes cluster, same result...
However, I have not tested the most recent version of Waves Node to determine if this bug is isolated to this image.

Possible quick solution to introduce stability is to disable `akka.jvm-exit-on-fatal-error`

**Comment by KardanovIR** [comment]
@bravetarget thank you for your report! This issue should be fixed with 409298d08aff536d5fdc060187c565ef7996697d. Can you check it again, please?

**Comment by bravetarget** [comment]
@KardanovIR my pleasure.  full usage test shows https://github.com/wavesplatform/node-docker-image/commit/409298d08aff536d5fdc060187c565ef7996697d resolved this issue, nice work! 👍
